responsibleai/environments/responsibleai-vision-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml

```diff
@@ -5,31 +5,22 @@ channels:
   - anaconda
   - pytorch
 dependencies:
-  - python=3.8
+  - python=3.7
```

Comment:
why are we reducing python to 3.7? also yaml file has py138 in its name

Reply:
due to automl dependency